packages/ui/core-components/src/lib/molecules/dimension-grid/_DimensionGrid.svelte

```diff
@@ -17,7 +17,7 @@
 	/** @type {string} */
 	export let name;
 
-	$: dimensions = data?.columns?.filter((col) => col.column_type === 'VARCHAR');
+	$: dimensions = data?.columns?.filter((col) => col.column_type === 'VARCHAR' || 'DOUBLE');
```

Comment:
Filtering for varchar columns is the intended behaviour.

Reply:
Thank You, reverted back to filter VARCHAR only.

Clarification:
So in the dimension grid example above with Fare Price and Airline, without the "DOUBLE" included filtering, the desired result is only to display Air line?

Reply:
Yeah exactly